pkgs/test_core/lib/src/runner/configuration/args.dart

```diff
@@ -118,25 +119,24 @@ final ArgParser _parser = (() {
     reporterDescriptions[reporter] = allReporters[reporter]!.description;
   }
 
-  parser.addSeparator('======== Output');
+  parser.addSeparator('Output:');
   parser.addOption('reporter',
       abbr: 'r',
-      help: 'The runner used to print test results.',
+      help: 'Set how to print test results.',
       defaultsTo: defaultReporter,
       allowed: reporterDescriptions.keys.toList(),
       allowedHelp: reporterDescriptions);
   parser.addOption('file-reporter',
-      help: 'The reporter used to write test results to a file.\n'
+      help: 'Set the reporter used to write test results to a file.\n'
           'Should be in the form <reporter>:<filepath>, '
-          'e.g. "json:reports/tests.json"');
+          'Example: "json:reports/tests.json"');
   parser.addFlag('verbose-trace',
-      negatable: false,
-      help: 'Whether to emit stack traces with core library frames.');
+      negatable: false, help: 'Emit stack traces with core library frames.');
```

Comment:
Do you know why this isn't negatable? This came up on the doc.

Reply:
I found the PR that added the flag: https://github.com/dart-lang/test/commit/b61a5362bb37eb7a9ffa6ccb2f7837d79701b4c6

Doesn't have a reason though.